content/teaching-curriculum/chapter1_high_standards

```diff
@@ -0,0 +1,32 @@
+---
+_db_id: 
```

Comment:
Non standard file name. the script wont be able to understand this.

Please make sure you follow the file naming conventions. Look at the directory names and the file names. There is a very clear pattern